Bixal/move.mil
# Feature 157850990 fixed one third two third wysiwyg layout

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- includes specificity to target our paragraph--text-with-image to match our live site

## Testing

To verify the changes proposed in this pull request…

1. pull code locally
2. at theme root, npm run build
3. make cr
4. navigate to /moving-guide, you should see the following layout for our paragraph--text-with-image

**Desktop** (stage left| diff right)
![desktop](https://user-images.githubusercontent.com/37077057/40853810-62e99784-659d-11e8-8513-fafd01d52f8a.gif)

**Mobile** (stage left| diff right)
![mobile](https://user-images.githubusercontent.com/37077057/40853824-66517f72-659d-11e8-80d4-b283644055de.gif)
